docker/entrypoint.sh

```diff
@@ -57,6 +57,12 @@ case "$1" in
             ls -lhR "$LIGHTWALLETD_DATA_DIR/db" || (echo "No $LIGHTWALLETD_DATA_DIR/db"; ls -lhR "$LIGHTWALLETD_DATA_DIR" | head -50 || echo "No $LIGHTWALLETD_DATA_DIR directory")
         elif [[ "$TEST_LWD_UPDATE_SYNC" -eq "1" ]]; then
             # Starting with a cached Zebra and lightwalletd tip, run a quick update sync.
+            #
+            # Temporarily skip this test, because the lwd cached state doesn't work.
+            # TODO: re-enable this test (#4415)
+            echo "Test skipped due to bug #4415"
+            exit 0
```

Comment:
@gustavovalverde did you want to merge PR #4486 as the permanent fix, or this PR as a temporary fix?

(Feel free to close this PR if #4486 works.)

Reply:
I'd prefer to wait for a complete successful run in https://github.com/ZcashFoundation/zebra/pull/4486 if that happens, then let's merge that one and close this one.